docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -274,6 +293,27 @@ To add new subscription rules and edit existing ones, complete the following ste
 
 If you plan to use the API to add products to the subscription sales channel, see [product channel assignments](/api-docs/multi-storefront/api-guide#products) for more information.
 
+## Deploying your app with Vercel
+
+The BigCommerce Subscription Foundation framework enables you to deploy your application with Vercel directly from the GitHub repo.
+
+To deploy your app with Vercel:
+
+1. Login to GitHub, and then navigate to the [BigCommerce Subscription Foundation README.md](https://github.com/bigcommerce/subscription-foundation/blob/main/README.md) file.
+2. Scroll down the page to the **Deploy with Vercel** section and click **Deploy**.
+3. In the **Get started** section, select a Git provider, and then authorize Vercel to connect to your provider.
+4. In the **Create Git Repository**, select a Git scope, enter a name for the repository, and then click **Create**. 
+5. (Optional) If you are deploying to Vercel from a GitHub organization, you need to create a team by following the steps in the **Create a team** section.
+6. In the **Configure Project** section, enter the connection string for the Supabase database you created in the previous step in the **DATABASE_URL** field. See [Run migration and start the server](#run-migration-and-start-the-server) for information on creating the database and obtaining the `DATABASE_URL`.
+7. Enter '1234' for the remaining environment variables and then click **Deploy**. The deployment process will take a few minutes.
+8. After you have successfully deployed Vercel, it is recommended that you update your environment variables. For more information, see [Declare environment variables](#declare-environment-variables). 
```

Comment:
```suggestion
8. After you have successfully deployed to Vercel, it is recommended that you update your environment variables. For more information, see [Declare environment variables](#declare-environment-variables). 
```
More than recommended, if you want the app to integrate properly then you must go back and enter in the rest of them now.